modules/raycast/raycast_occlusion_cull.cpp

```diff
@@ -570,16 +569,16 @@ Projection RaycastOcclusionCull::_jitter_projection(const Projection &p_cam_proj
 		} break;
 	}
 
-	// The multiplier here determines the divergence from center,
-	// and is to some extent a balancing act.
-	// Higher divergence gives fewer false hidden, but more false shown.
+	jitter *= Vector2(p_half_extents.x / (float)p_viewport_size.x, p_half_extents.y / (float)p_viewport_size.y);
+
+	// The multiplier here determines the jitter magnitude in pixels.
+	// It seems like a value of 0.66 matches well the above jittering pattern as it generates subpixel samples at 0, 1/3 and 2/3
+	// Higher magnitude gives fewer false hidden, but more false shown.
 	// False hidden is obvious to viewer, false shown is not.
 	// False shown can lower percentage that are occluded, and therefore performance.
-	jitter *= Vector2(1 / (float)p_viewport_size.x, 1 / (float)p_viewport_size.y) * 0.9f;
+	jitter *= 0.66;
```

Comment:
`0.66f` will do 32 bit calculation on 32 bit. 64 bit calculation not needed on 32 bit I don't think.